package.json

```diff
@@ -81,29 +81,28 @@
     "ui:lint:fix": "stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-ui/**/*.scss\"",
-    "core:start": "npm-run-all core:build -p core:build:sass:watch 'core:build:ts -- -w' 'ng serve dev-core'",
-    "core:dev:build": "ng build dev-core --aot --sourceMap=false",
-    "core:dev:build:prod": "ng build dev-core --prod",
-    "core:lint": "npm run prelint && tslint -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" -e \"src/clr-core/**/*.css.ts\" && tslint -p . -c ./src/dev-core/tslint.json \"src/dev-core/**/*.ts\" && stylelint --config ./src/clr-core/.stylelintrc \"src/clr-core/**/*.scss\" \"src/dev-core/**/*.scss\"",
-    "core:lint:fix": "npm run prelint && tslint --fix -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" && tslint --fix -p . -c ./src/dev-core/tslint.json \"src/dev-core/**/*.ts\" && stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-core/**/*.scss\" \"src/dev-core/**/*.scss\"",
+    "core:start": "npm-run-all core:build -p core:build:watch storybook",
+    "core:start:old": "npm-run-all core:build -p core:build:watch 'ng serve dev-core'",
+    "storybook": "start-storybook -s ./src/clr-core/.storybook/public -p 6006 --config-dir=./src/clr-core/.storybook",
+    "core:dev:build": "build-storybook -s ./src/clr-core/.storybook/public -c src/clr-core/.storybook -o dist/dev-core",
+    "core:lint": "npm run prelint && tslint -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" -e \"src/clr-core/**/*.css.ts\" && stylelint --config ./src/clr-core/.stylelintrc \"src/clr-core/**/*.scss\"",
+    "core:lint:fix": "npm run prelint && tslint --fix -p . -c ./src/clr-core/tslint.json \"src/clr-core/**/*.ts\" stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-core/**/*.scss\"",
     "core:test": "npm-run-all -p core:test:treeshaking core:test:unit",
     "core:test:watch": "ng test clr-core --watch --code-coverage",
     "core:test:unit": "ng test clr-core --watch=false",
     "core:test:treeshaking": "webpack --config src/clr-core/test-bundles/webpack.config.js && bundlesize --config src/clr-core/test-bundles/bundlesize.json",
     "core:build": "npm-run-all core:build:sass -p core:build:package 'core:build:ts -- --force' core:build:api core:build:clean",
-    "core:build:clean": "del ./dist/clr-core/**/*.{tsbuildinfo,ngsummary.json} && del ./dist/clr-core/*.{tsbuildinfo,ngsummary.json}",
+    "core:build:watch": "npm-run-all -p core:build:sass:watch 'core:build:ts -- -w'",
+    "core:build:clean": "node ./src/clr-core/package.js",
     "core:build:package": "cpy '**/package.json' '../../dist/clr-core/' --cwd=src/clr-core --parents && cpy ./src/clr-core/README.md ./dist/clr-core/; replace '@VERSION' $npm_package_version ./dist/clr-core/package.json",
     "core:build:ts": "tsc --b src/clr-core/tsconfig.project.json",
-    "core:build:src": "npm-run-all core:build:sass:components",
+    "core:build:src": "npm-run-all core:build:sass:components 'core:build:ts -- --force'",
```

Comment:
typo (first `--`)?

Reply:
This is a bash syntax I think? In the run-all docs it described it as a way to pass a flag into another flag. The `core:build:ts` is technically a flag value of npm-run-all so its a way to pass `--force` down into `core:build:ts`. This was nice since I didnt have to write an additional npm script to just have a flag on.

Reply:
Ah interesting. I wasn't sure if that was intentional. Thanks for explanation! 👍